Tests/dep/VersionTests.swift

```diff
@@ -47,70 +48,42 @@ class VersionTests: XCTestCase, XCTestCaseProvider {
     }
 
     func testComparable() {
-        do {
-            let v1 = Version(1,2,3)
-            let v2 = Version(2,1,2)
+        // Precedence tests taken directly from http://semver.org (with a few additions)
+        var tests = [
+            Version("1.0.0-alpha"),
+            Version("1.0.0-alpha.1"),
+            Version("1.0.0-alpha.beta"),
+            Version("1.0.0-beta"),
+            Version("1.0.0-beta.2"),
+            Version("1.0.0-beta.11"),
+            Version("1.0.0-rc.1"),
+            Version("1.0.0"),
+            Version("1.0.1"),
+            Version("1.2.3"),
+            Version("2.1.2"),
+            Version("2.1.3"),
+            Version("2.2.2")
+        ].map{ $0! }
+        
+        var v1: Version = tests.removeFirst()
+        for v2 in tests {
             XCTAssertLessThan(v1, v2)
             XCTAssertLessThanOrEqual(v1, v2)
             XCTAssertGreaterThan(v2, v1)
             XCTAssertGreaterThanOrEqual(v2, v1)
             XCTAssertNotEqual(v1, v2)
-
+            
             XCTAssertLessThanOrEqual(v1, v1)
             XCTAssertGreaterThanOrEqual(v1, v1)
             XCTAssertLessThanOrEqual(v2, v2)
             XCTAssertGreaterThanOrEqual(v2, v2)
-        }
-
-        do {
-            let v3 = Version(2,1,3)
-            let v4 = Version(2,2,2)
-            XCTAssertLessThan(v3, v4)
-            XCTAssertLessThanOrEqual(v3, v4)
-            XCTAssertGreaterThan(v4, v3)
-            XCTAssertGreaterThanOrEqual(v4, v3)
-            XCTAssertNotEqual(v3, v4)
-
-            XCTAssertLessThanOrEqual(v3, v3)
-            XCTAssertGreaterThanOrEqual(v3, v3)
-            XCTAssertLessThanOrEqual(v4, v4)
-            XCTAssertGreaterThanOrEqual(v4, v4)
-        }
-
-        do {
-            let v5 = Version(2,1,2)
-            let v6 = Version(2,1,3)
-            XCTAssertLessThan(v5, v6)
-            XCTAssertLessThanOrEqual(v5, v6)
-            XCTAssertGreaterThan(v6, v5)
-            XCTAssertGreaterThanOrEqual(v6, v5)
-            XCTAssertNotEqual(v5, v6)
-
-            XCTAssertLessThanOrEqual(v5, v5)
-            XCTAssertGreaterThanOrEqual(v5, v5)
-            XCTAssertLessThanOrEqual(v6, v6)
-            XCTAssertGreaterThanOrEqual(v6, v6)
-        }
-
-        do {
-            let v7 = Version(0,9,21)
-            let v8 = Version(2,0,0)
-            XCTAssert(v7 < v8)
-            XCTAssertLessThan(v7, v8)
-            XCTAssertLessThanOrEqual(v7, v8)
-            XCTAssertGreaterThan(v8, v7)
-            XCTAssertGreaterThanOrEqual(v8, v7)
-            XCTAssertNotEqual(v7, v8)
-
-            XCTAssertLessThanOrEqual(v7, v7)
-            XCTAssertGreaterThanOrEqual(v7, v7)
-            XCTAssertLessThanOrEqual(v8, v8)
-            XCTAssertGreaterThanOrEqual(v8, v8)
```

Comment:
Thanks, I'm sure there aren't bugs with your stuff, but generally with tests I feel more is better and keeping them makes me feel safer.